### ISSUE TITLE
Radius marker causing fps issues.

### ISSUE THREAD

**Issue by emiljensen2** [Skretzo/runelite-plugins#27]
Radius marker has started causing fps issues/stutters, it gets worse the longer you play, a few of my clanmates have reported the same issues and disabling radius marker fixes the issue.
At one point it got so bad my akkha het puzzle dropped down to like 15 frames, toggling it off causes it to return to normal, but toggling it back on it goes back to 15 fps.
Runelite 1.10.9.1
launcher 2.5.0
Radius marker 1.8

**Comment by Skretzo** [comment]
Were you/they by any chance also using the Better NPC Highlight plugin? That plugin had an FPS drop issue that was resolved 18 minutes after you posted this issue. If you think you/they used that plugin, it would be very helpful if you could check if the issue still occurs, either by eliminating said plugin or restarting the client to fetch any new plugin versions.

**Comment by emiljensen2** [comment]
Yeah majority of us are, tho I think theirs is just a general lowering of fps and has been an issue for a bit, I'll have a look, tho the last few days whenever someone was recommended to try disabling radius marker it helped. I think there's been like 3 or 4 people in the clan when I've been there.
My client log ends in a java heap space at 2023-08-01 21:54:15 which is in akkha right after the het puzzle (my client crashed)
It doesn't really look like there's any errors in particular about radius markers in my log, I've attached it below. I played using the iron profile, I had done tob, completed the dt2 quest, done a few vard kills, did some toa in this session.
[client - radius.log](https://github.com/Skretzo/runelite-plugins/files/12245848/client.-.radius.log)
